img/COPYING

```diff
@@ -5,3 +5,11 @@
 ## leave_call.ogg
 @author Marco Ambrosini (marcoambrosini@icloud.com) 
 @license CC0-1.0
+
+## join_call.flac
```

Comment:
you add 3 files but only 2 licenses?

Reply:
We did not specify a license for `LibremPhoneCall.ogg` before, so I did not add it for the FLAC version.

Reply:
@marcoambrosini any idea about the third license ?

Reply:
No, that file was already there when I added the join and leave call sounds.